app/views/estimate_flow/forms/_housing_costs.html.slim

```diff
@@ -0,0 +1,59 @@
+= form_for(@form, url: wizard_path, method: :put) do |form|
+  = form.govuk_error_summary t("generic.error_summary_title")
+
+  ruby:
+    partner_key = if @check.partner
+                    "heading_hint_partner"
+                  else
+                    "heading_hint"
+                  end
+
+  = render "shared/heading",
+          header_text: t("estimate_flow.#{i18n_key}.heading"),
+          post_header_texts: t("estimate_flow.#{i18n_key}.#{partner_key}")
+  h2.govuk-heading-m class="govuk-!-margin-bottom-2"
+     = t("estimate_flow.#{i18n_key}.housing_payments.label")
+  p.govuk-caption-m class="govuk-!-margin-bottom-4"
+    - t("estimate_flow.#{i18n_key}.housing_payments.hint_paragraph").each
+      p.govuk-caption-m class="govuk-!-margin-bottom-2" = _1
+
+  = render "shared/money_input",
+          form:,
+          field: :housing_payments,
+          width: 10,
+          label_text: t("estimate_flow.#{i18n_key}.value")
+  = form.govuk_collection_radio_buttons :housing_payments_frequency,
+                                        @form.housing_payment_frequencies,
+                                        :first,
+                                        :last,
+                                        legend: -> { tag.p(t("estimate_flow.#{i18n_key}.frequency")) }
+  h2.govuk-heading-m class="govuk-!-margin-bottom-2"
+     = t("estimate_flow.#{i18n_key}.housing_benefits.label")
+  p.govuk-caption-m class="govuk-!-margin-bottom-4"
+    - t("estimate_flow.#{i18n_key}.housing_benefits.hint_paragraph").each
+      p.govuk-caption-m class="govuk-!-margin-bottom-2" = _1
+  ul.govuk-list.govuk-list--bullet class="govuk-hint govuk-!-margin-top-0 govuk-!-margin-bottom-0"
```

Comment:
Ok, this section needs tightening up. What you want is something like:

```
ul.govuk-list.govuk-list--bullet
  li = t("estimate_flow.#{i18n_key}.housing_benefits.hint_bullet_1")
  - if @check.partner
    li = t("estimate_flow.#{i18n_key}.housing_benefits.hint_bullet_2_partner")
  li = t("estimate_flow.#{i18n_key}.housing_benefits.hint_bullet_3")
```
To do this, you'll need to change the yaml for the bullets from:
```
hint_bullet_1:
  - Blah
 ```
to:
```
hint_bullet_1: Blah
 ```

Because each bullet is just one bullet, not an array of bullets.